recruiter-email.txt

```diff
@@ -0,0 +1,14 @@
+<Name>,
+
+Just wanted to reach out about your first CoffeeOps meet-up last Friday.
```

Comment:
Suggestion:

Just wanted to reach out about _**<your first|the next>**_ CoffeeOps meet-up _**on \<date\>**_.